src/guide/typescript-support.md

```diff
@@ -4,7 +4,7 @@
 
 ## Official Declaration in NPM Packages
 
-A static type system can help prevent many potential runtime errors as applications grow, which is why Vue 3 is written in TypeScript. This means you don't need any additional tooling to use TypeScript with Vue - it has a first-class citizen support.
+A static type system can help prevent many potential runtime errors as applications grow, which is why Vue 3 is written in TypeScript. This means you don't need any additional tooling to use TypeScript with Vue - it has a first-class citizens' support.
```

Comment:
I agree that the current wording is incorrect but only option 2 sounds right to me:

> it has first-class citizen support.

Either that or a total rewording.

Reply:
Option 2 sounds alright to me.